AdblockPlusSafari/AdblockPlusController.m

```diff
@@ -192,3 +208,67 @@ - (void)updateAccessoryViewOfCell:(UITableViewCell *)cell
 }
 
 @end
+
+@implementation AdblockPlusController: AdblockPlusControllerBase
+#ifdef CONFIGURABLE_CUSTOM_FILTER_LIST_ENABLED
+
+#pragma mark - UITableViewDataSource
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
+{
+  if (section == AdblockPlusControllerSectionDefault) {
+    return [super tableView:tableView numberOfRowsInSection:section] + 1;
+  } else {
+    return [super tableView:tableView numberOfRowsInSection:section];
+  }
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+  if (indexPath.section == AdblockPlusControllerSectionDefault && indexPath.row > 0) {
```

Comment:
This block has a few `else` after `return` - I think the code can be simplified a bit.